builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -35,6 +35,7 @@ import groovy.transform.Field
 @Field def service_config
 @Field def websiteType
 @Field def azureAccount
+@Field def angularVersion
```

Comment:
Defining this as a global var can affect builds on other user apps. Let's scope it to local by default.

Reply:
Done. Now passed as local variable